app/views/api/v2/hosts/main.json.rabl

```diff
@@ -37,6 +37,13 @@ node :hostgroup_name do |host|
   host.hostgroup.name if host.hostgroup.present?
 end
 
+node :created_by_user do |host|
+  if host.created_by_user_id.present?
+    user = User.find(host.created_by_user_id)
+    "by #{user.firstname} #{user.lastname}"
```

Comment:
Works for me, any chance there's also `User.username`?

Reply:
except for the "by" string that should be part of the frontend imho

Reply:
> username

I didn't find any shortcut for that (username didn't work).